build-tools/src/main/kotlin/org/jetbrains/kotlin/ExecutorService.kt

```diff
@@ -311,3 +320,233 @@ private fun sshExecutor(project: Project) : ExecutorService = object : ExecutorS
         }
     }
 }
+
+private fun deviceLauncher(project: Project) = object : ExecutorService {
+    private val xcProject = Paths.get(project.testOutputRoot, "launcher")
+
+    private val idb = project.findProperty("idb_path") as? String ?: "idb"
+
+    private val deviceName = project.findProperty("device_name") as? String
+
+    override fun execute(action: Action<in ExecSpec>): ExecResult? {
+        val udid = targetUDID()
+        println("Found device UDID: $udid")
+        install(udid, "build/KonanTestLauncher.ipa")
```

Comment:
Understanding a location of this `build` directory is complicated. May be specify the path more explicitly, say `xcProject.resolve("KonanTestLauncher/build/KonanTestLauncher.ipa")`?